new survey form

Sharifarezaie opened the pull request:
This the new version of Survey Form project.

Sharifarezaie committed:
new survey form

Sharifarezaie commented:
@ahzamir I have solved the linter conflict of survey form project please check it and inform me i case of any issue.

ahzamir commented:
@Sharifarezaie 

There is no linters config in this project and besides that you only have one commit on this project which is not acceptable.

Sharifarezaie committed:
i add .github

Sharifarezaie commented:
@farida faqiri
salam farid  lotfan projeck ma chec

Sharifarezaie commented:
@ahzamir Hi sir pleas check my survey- form

ahzamir commented:
@Sharifarezaie 

- Still, i can not see your partner's feedback on your code, and besides that, you still have fewer commits on your project you need to add at least a few more commits like 4-5 so I can check your project:
  - Change the design or add some design to your project and have some more commits to your project.

Sharifarezaie commented:
@Farida Faqiri
pleas check my projects

Sharifarezaie commented:
@farida faqiri
salam farida jan pleas project ma check 
wa fedback bedy

Sharifarezaie committed:
background color update

Sharifarezaie committed:
submit button color changed

Sharifarezaie committed:
Update README.md

# Survey form  🚀

## Project Description 📝

> This is  my surve form project
> ITt is a servey form for getting information about intrest of student about the classes and whether they have computer or no.



## Demo 📸

![Demo](file:///C:/Users/Rezaie/Desktop/project/NewRepo/index.html)

## Technologies Used 🛠️

- HTML
- CSS

## Installation 💻

Folow these steps to set up the project on your local machine 
Ensure you have the following software installed on your machine

bash
#git clone


bash
#git@github.com:Sharifarezaie/NewRepo/.git


## Usage 🎯

you can access to my project through git clone 
-git clone the HTTP or My SSH code link from the repository you wat to clone 

 
#git clone


## Features ⭐

1: General Information
Name:
Email:
Age:
Gender:
2: Kitchen Use and Preferences
3: Food Taste Preferences
4: Suggestions and Feedback


## Author 👩‍💻
Sharifa Rezaie
- Email: [example](sharifashaida82@gmail.com)

## Contributing 🤝

We welcome contributions from the community! Please follow the guidelines below to contribute to our project.
<br />
1: Fork the Repository: Click the "Fork" button at the top right of the repository page to create a copy of the repository on your GitHub account.
<br />
2: Clone Your Fork: Clone your forked repository to your local machine using the following command:git clone https://github.com/yourusername/yourprojectname.git
<br />
3: Create a Branch: Create a new branch for your work to ensure your changes are isolated from the main branch:git checkout -b feature-or-bugfix-description
<br />
4: Make Your Changes: Implement your changes, whether fixing a bug or adding a new feature. Be sure to follow the project's coding standards and guidelines.
<br />
5: Commit Your Changes: Commit your changes with a descriptive commit message:git add .
git commit -m "Description of the changes"
<br />
6: Push to Your Fork: Push your changes to your forked repository:git push origin feature-or-bugfix-description
<br />
7: Submit a Pull Request: Go to the original repository and submit a pull request from your forked repository. Provide a clear and concise description of your changes and why they should be merged..




- HTML
- CSS

## Installation 💻

Folow these steps to set up the project on your local machine 
Ensure you have the following software installed on your machine

bash
#git clone


bash
#(https://github.com/Sharifarezaie/NewRepo.git)


## Usage 🎯

you can access to my project through git clone 
-git clone the HTTP or My SSH code link from the repository you wat to clone 

 
#git clone


## Features ⭐

1: General Information
Name:
Email:
Age:
Gender:
2: Kitchen Use and Preferences
3: Food Taste Preferences
4: Suggestions and Feedback
## Author 👩‍💻


Fatima Abdullahi

- Email: [ fatima333abdullahi@gmail.com)


## Contributing 🤝

We welcome contributions from the community! Please follow the guidelines below to contribute to our project.
<br />
1: Fork the Repository: Click the "Fork" button at the top right of the repository page to create a copy of the repository on your GitHub account.
<br />
2: Clone Your Fork: Clone your forked repository to your local machine using the following command:git clone https://github.com/yourusername/yourprojectname.git
<br />
3: Create a Branch: Create a new branch for your work to ensure your changes are isolated from the main branch:git checkout -b feature-or-bugfix-description
<br />
4: Make Your Changes: Implement your changes, whether fixing a bug or adding a new feature. Be sure to follow the project's coding standards and guidelines.
<br />
5: Commit Your Changes: Commit your changes with a descriptive commit message:git add .
git commit -m "Description of the changes"
<br />
6: Push to Your Fork: Push your changes to your forked repository:git push origin feature-or-bugfix-description
<br />
7: Submit a Pull Request: Go to the original repository and submit a pull request from your forked repository. Provide a clear and concise description of your changes and why they should be merged..

Sharifarezaie committed:
Update README.md

# Survey form  🚀

## Project Description 📝

> This is  my surve form project
> ITt is a servey form for getting information about intrest of student about the classes and whether they have computer or no.



## Demo 📸

![Demo](file:///C:/Users/Rezaie/Desktop/project/NewRepo/index.html)

## Technologies Used 🛠️

- HTML
- CSS

## Installation 💻

Folow these steps to set up the project on your local machine 
Ensure you have the following software installed on your machine

bash
#git clone


bash
#git@github.com:Sharifarezaie/NewRepo/.git


## Usage 🎯

you can access to my project through git clone 
-git clone the HTTP or My SSH code link from the repository you wat to clone 

 
#git clone


## Features ⭐

1: General Information
Name:
Email:
Age:
Gender:
2: Kitchen Use and Preferences
3: Food Taste Preferences
4: Suggestions and Feedback


## Author 👩‍💻
Sharifa Rezaie
- Email: [example](sharifashaida82@gmail.com)

## Contributing 🤝

We welcome contributions from the community! Please follow the guidelines below to contribute to our project.
<br />
1: Fork the Repository: Click the "Fork" button at the top right of the repository page to create a copy of the repository on your GitHub account.
<br />
2: Clone Your Fork: Clone your forked repository to your local machine using the following command:git clone https://github.com/yourusername/yourprojectname.git
<br />
3: Create a Branch: Create a new branch for your work to ensure your changes are isolated from the main branch:git checkout -b feature-or-bugfix-description
<br />
4: Make Your Changes: Implement your changes, whether fixing a bug or adding a new feature. Be sure to follow the project's coding standards and guidelines.
<br />
5: Commit Your Changes: Commit your changes with a descriptive commit message:git add .
git commit -m "Description of the changes"
<br />
6: Push to Your Fork: Push your changes to your forked repository:git push origin feature-or-bugfix-description
<br />
7: Submit a Pull Request: Go to the original repository and submit a pull request from your forked repository. Provide a clear and concise description of your changes and why they should be merged..

ahzamir commented:
@Sharifarezaie 

- Your project has a Conflict solve it first then ask your partner for feedback and also you mentioned a wrong person on your pull request

Sharifarezaie commented:
@far_faqiri
salam farida jan lotfan projeck ma chek koned

ahzamir commented:
@Sharifarezaie 

- Your project has a conflict search on google or ask your partner to help you solve this.

Sharifarezaie commented:
استاد لطفا  تمام  ایرر های پروژه مارا یکبار برای ما بیان کنید
تا بتوانیم همزمان تمام ایرر را حل کنیم  تا بخاطر فیدبک دوباره ساعت ها
منتظر  نباشیم   درظمن پارتنر مه مینشن میکنم میگه که هیج پیامی نیامده  وقتی
پیام به شما می آید   امکان نداره به اونا  ارسال نشه  درحال که نام اکونت
GitHub  خوده برایم داده!

On Fri, Jun 7, 2024, 10:36 PM Ahmad Zamir Yousufi ***@***.***>
wrote:

> @Sharifarezaie <https://github.com/Sharifarezaie>
>
>    - Your project has a conflict search on google or ask your partner to
>    help you solve this.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/Sharifarezaie/NewRepo/pull/1#issuecomment-2155291734>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BIHKJSLUYDYK7TFCBNEXTXTZGHZA7AVCNFSM6AAAAABIUEJ47KVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDCNJVGI4TCNZTGQ>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

ahzamir commented:
@Sharifarezaie 

- Your project has conflict issues and I can not provide feedback if you don't solve that first.
- You used the incorrect username of your partner this is the correct one:
@faridafaqiri 
- Help her solve the conflict issue.
